README.md

```diff
@@ -1,7 +1,20 @@
-# `twist_to_ackermann` package
+# twist_to_ackermann
 
-This package contains a node that converts incoming `std_msgs/Twist` messages to `ackermann_msgs/AckermannDriveStamped`
-messages.
+twist_to_ackermann is a new repository that was developed to translate ROS 2 messages from desired velocities (std_msgs/msg/Twist) to steering and velocity commands for an Ackermann vehicle (ackermann_msgs/msg/AckermannDriveStamped). 
 
-See [Planning for car-like robots](https://wiki.ros.org/teb_local_planner/Tutorials/Planning%20for%20car-like%20robots)
```

Comment:
I would keep this so we can acknowledge what we based our implementation off of

Reply:
Reverted this line